### PR TITLE
added first version of settings bundle

### PR DESCRIPTION
**Pull request by yimenglovesai**
https://github.com/Loopring/loopr-ios/issues/13

added first version of settings bundle which includes app version and build number

### CODE REVIEW

File: loopr-ios.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2928,11 +2936,11 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 47TZG75K6F;
```
Comment (xiaowheat) [inline loopr-ios.xcodeproj/project.pbxproj]: Could you remove the development team in the project settings.

File: loopr-ios.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2928,11 +2936,11 @@
...
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = "loopr-ios/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "org.loopring.alpha.loopr-ios";
+				PRODUCT_BUNDLE_IDENTIFIER = "org.loopring.loopr-ios";
```
Comment (xiaowheat) [inline loopr-ios.xcodeproj/project.pbxproj]: Could you revert the product bundle identifier?

File: loopr-ios/SettingsBundleHelper.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+//  loopr-ios
+//
+//  Created by yimenglovesai on 4/22/18.
+//  Copyright © 2018 Loopring. All rights reserved.
+//
+
+import Foundation
+
+class SettingsBundleHelper {
+    struct SettingsBundleKeys {
```
Comment (xiaowheat) [inline loopr-ios/SettingsBundleHelper.swift]: Is SettingBundleKeys used in any place？

File: loopr-ios/SettingsBundleHelper.swift [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        static let BuildVersionKey = "build_preference"
+        static let AppVersionKey = "version_preference"
+    }
+
+    class func setVersionAndBuildNumber() {
+        
+        let version = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String
+        let build = Bundle.main.object(forInfoDictionaryKey: kCFBundleVersionKey as String) as! String
+
+        UserDefaults.standard.set(version, forKey: "app_version")
```
Comment (xiaowheat) [inline loopr-ios/SettingsBundleHelper.swift]: Please add to ```UserDefaultsKeys```  Example: 
```
let defaults = UserDefaults.standard
let encodedData = NSKeyedArchiver.archivedData(withRootObject: appWallets)
defaults.set(encodedData, forKey: UserDefaultsKeys.appWallets.rawValue)
```

Comment (yimenglovesai) [reply loopr-ios/SettingsBundleHelper.swift]: updated.

**Pull request merged**